parlai/chat_service/services/terminal_chat/playground/server.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import socket
```

Comment:
I don't like that all of this is living as just global state in a module. It seems more appropriate to write this into a class with like a serve_forever() function. You can look at python's builtin `http.server` for inspiration.

Reply:
@stephenroller Thanks for your suggestion and feedback we will look into it. However, this code was just for me and my teammate to experiment and learn about how the socket communication works and hence is essentially "rough-work". As I mentioned in the README in this dir, it's just a toy example and would be removed before this PR is merged.